File: vcr-sample/app/services/star_wars.rb
require 'net/http'

class StarWars
    def self.get_starwars_character
        uri = URI.parse('https://swapi.dev/api/people/2')
        response = Net::HTTP.get_response(uri)
        raise StandardError, "Failed! #{response.code}" unless response.code == '200'
        JSON.parse(response.body)
    end
end

File: server/src/services/star-wars.ts
import axios, { type AxiosResponse } from 'axios'

export async function getStarWarsCharacter (): Promise<AxiosResponse> {
  return await axios({
    method: 'get',
    url: 'https://swapi.dev/api/people/2'
    // responseType: 'stream'
  })
}
